ccd-definition/ComplexTypes/CareSupervision/C2DocumentBundle-nonprod.json

```diff
@@ -0,0 +1,85 @@
+[
```

Comment:
Just double checking but this is just a reordering with a couple of fields renamed?

Reply:
yep, couple of fields renamed, and tab fields reordered